src/olmo_core/internal/experiment.py

```diff
@@ -411,7 +413,22 @@ def build_config(
 def launch(config: ExperimentConfig):
     log.info(config)
     assert config.launch is not None
-    config.launch.launch(follow=True, launch_timeout=5 * 60)
+
+    # Only send local Slack notifications when slack callback is enabled.
+    slack_enabled = False
+    for callback in config.trainer.callbacks.values():
+        if isinstance(callback, SlackNotifierCallback):
+            if callback.enabled and SLACK_WEBHOOK_URL_ENV_VAR in os.environ:
+                slack_enabled = True
+            break
+
+    config.launch.launch(
+        follow=True,
+        slack_notifications=slack_enabled,
+        launch_timeout=5 * 60,
+        #  step_timeout=30 * 60,  # hard timeout kills the job
+        # step_soft_timeout=10 * 60,  # soft timeout only sends slack warning
```

Comment:
Can we no longer set these? It would be nice if we could set these anyways, so when we decide to turn this on, we get the behavior we're used to.